multimodal/src/autogluon/multimodal/data/process_image.py

```diff
@@ -403,11 +413,11 @@ def process_one_sample(
         column_start = 0
         if self.prefix.lower().startswith(MMLAB_MODELS):
             for per_col_name, per_col_content in image_paths.items():
-                if per_col_name != "rois":  # TODO: remove hardcode
-                    mm_data["img_info"] = dict(filename=per_col_content[0])
-                else:
+                if is_rois_input(per_col_content):
```

Comment:
We should rely on the infer_type logic.

Reply:
The type inferred by `infer_type` is not forwarded to `process_image`, but here we use the same function that `infer_type` is using. Also we will move these code to `process_rois.py` later.